packages/govuk-frontend/src/govuk/assets/manifest.webmanifest

```diff
@@ -0,0 +1,6 @@
+{
+  "icons": [
+    { "src": "images/govuk-android-icon.png", "type": "image/png", "sizes": "192x192" },
+    { "src": "images/govuk-android-icon-512.png", "type": "image/png", "sizes": "512x512" }
```

Comment:
Coming through on Chrome macOS

<img width="398" alt="Icons from manifest" src="https://github.com/alphagov/govuk-frontend/assets/415517/add64bf3-ba38-4186-ab2d-f4238aced351">

Reply:
Although this is where it gets tricky as a "copy and paste" manifest, dynamic fields:

<img width="509" alt="Validation errors" src="https://github.com/alphagov/govuk-frontend/assets/415517/2f502df4-feb9-4efe-aff4-9d349daf9db9">

Reply:
Likewise in Firefox's inspector 👍

Reply:
Not checked the Safari 17 beta yet, but "Add to Home Screen" will work on desktop too
https://webkit.org/blog/14205/news-from-wwdc23-webkit-features-in-safari-17-beta/

Assuming it's using manifest support, the name might need a tweak?

Or consolidating our "touch icons" for multiple platform sizes?

```suggestion
    { "src": "images/govuk-touch-icon.png", "type": "image/png", "sizes": "192x192" },
    { "src": "images/govuk-touch-icon-512.png", "type": "image/png", "sizes": "512x512" }
```

Reply:
The warnings under 'Installability' are only relevant to PWAs and aren't required if we're simply supplying icons for home screen and bookmarking purposes. 

I'd forgotten about PWAs coming to desktop Safari. I like to live dangerously so I was already running Safari 17, and have added a screenshot of the icons it used above. I'm not 100% sure (though I assume based on the documentation) that it uses he manifest icons if they're defined, so there's good reason to potentially rename them.

Reply:
Weirdly, since updating to the newest Sonoma beta over the weekend (presumably clearing out Safari caches in the process) it now appears to be favouring the apple-touch-icon over the manifest file. 

Not sure if that's an intentional change in the update, or whether caching fudged with the results before or after. May well be the latter—Safari is slightly notorious for aggressive caching of favicons.